hack/Dockerfile.unit

```diff
@@ -1,6 +1,10 @@
 FROM registry.hub.docker.com/library/golang:1.13
 
-WORKDIR /usr/local/kubebuilder
-COPY /hack/tools /usr/local/kubebuilder/
+COPY .  /CAPM3
+RUN cd /CAPM3 && go mod download -json
+RUN cp -r /CAPM3/hack/tools /usr/local/kubebuilder/
+RUN cd .. && rm -rf CAPM3
```

Comment:
please try to limit the number of layers that you will be generating. All this can be merged into a single RUN.

Reply:
okay